### DISCUSSION TITLE
What is the v4 release schedule?

### DISCUSSION THREAD

**Discussion by iCodeWoods** [microsoft/cpp_client_telemetry#1020] [General]
I have seen a lot of issues tagged with v4, and I am looking forward to the release of the v4 version. When will it be released? Don't need a specific date, just want to know the approximate, such as the middle of next month or something?

**Comment by lalitb** [comment]
There is no date planned as of now for v4. In general, v4 tag is associated with crash associated with SDK shutdown,  i.e, use-after-free scenario. This will eventually involve SDK redesign and probably API changes.
